sdk/client-core/core-http-okhttp/pom.xml

```diff
@@ -41,30 +41,30 @@
     <jacoco.min.linecoverage>0.80</jacoco.min.linecoverage>
     <jacoco.min.branchcoverage>0.75</jacoco.min.branchcoverage>
     <javaModulesSurefireArgLine>
-      --add-exports com.generic.core/com.generic.core.implementation.http=ALL-UNNAMED
-      --add-exports com.generic.core/com.generic.core.implementation.http.serializer=ALL-UNNAMED
-      --add-exports com.generic.core/com.generic.core.implementation.util=ALL-UNNAMED
+      --add-exports io.clientcore.core/io.clientcore.core.implementation.http=ALL-UNNAMED
+      --add-exports io.clientcore.core/io.clientcore.core.implementation.http.serializer=ALL-UNNAMED
+      --add-exports io.clientcore.core/io.clientcore.core.implementation.util=ALL-UNNAMED
 
-      --add-opens com.generic.core.http.okhttp/com.generic.core.http.okhttp=ALL-UNNAMED
-      --add-opens com.generic.core.http.okhttp/com.generic.core.http.okhttp.implementation=ALL-UNNAMED
+      --add-opens io.clientcore.core.http.okhttp/io.clientcore.core.http.okhttp=ALL-UNNAMED
+      --add-opens io.clientcore.core.http.okhttp/io.clientcore.core.http.okhttp.implementation=ALL-UNNAMED
     </javaModulesSurefireArgLine>
 
     <!-- Enables fail on deprecated API usage. -->
     <compiler.failondeprecatedstatus/>
 
     <javadoc.excludePackageNames>
-      com.generic.core.annotation,com.generic.core.credential,com.generic.core.http,com.generic.core.http.annotation,com.generic.core.http.client,
-      com.generic.core.http.exception,com.generic.core.http.models,com.generic.core.http.pipeline,com.generic.core.http.policy,com.generic.core.implementation,
-      com.generic.core.implementation*,com.generic.core.models,com.generic.core.util,com.generic.core.util*,com.generic.json,
+      io.clientcore.core.annotation,io.clientcore.core.credential,io.clientcore.core.http,io.clientcore.core.http.annotation,io.clientcore.core.http.client,
+      io.clientcore.core.http.exception,io.clientcore.core.http.models,io.clientcore.core.http.pipeline,io.clientcore.core.http.policy,io.clientcore.core.implementation,
+      io.clientcore.core.implementation*,io.clientcore.core.models,io.clientcore.core.util,io.clientcore.core.util*,io.clientcore.core.json,
```

Comment:
We should generate our javadoc at some point to ensure things look good.

Reply:
Will update in https://github.com/Azure/azure-sdk-for-java/issues/37526